ui/admin/package.json

```diff
@@ -35,7 +35,8 @@
     "posttest": "ember coverage-merge && yarn clean:coverage"
   },
   "dependencies": {
-    "ember-named-blocks-polyfill": "^0.2.5"
+    "ember-named-blocks-polyfill": "^0.2.5",
```

Comment:
Can you explain why we are adding these 2 dependencies to admin?

Reply:
I originally added `jsonlint` which is a package needed for the linter built into CodeMirror. However, this has been removed from `admin/package.json` and moved to `rose/package.json` to keep all CodeMirror related imports as close to Rose as possible.